### PR TITLE
Create module declaration for dragster

### PR DESCRIPTION
**Pull request by zskovacs**
- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [ ] Run `npm run lint package-name` if a `tslint.json` is present.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: 
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

### CODE REVIEW

File: dragster/index.d.ts [hunk 1]
```diff
@@ -3,10 +3,32 @@
...
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare class Dragster {
-    constructor(elem: HTMLElement);
-    removeListeners(): void;
-    reset(): void;
+// Type definitions for dragster 0.1
+// Project: https://github.com/bensmithett/dragster
+// Definitions by: Zsolt Kovacs <https://github.com/zskovacs>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```
Comment (DanielRosenwasser) [inline dragster/index.d.ts]: This file shouldn't have two headings.

File: dragster/index.d.ts [hunk 2]
```diff
@@ -3,10 +3,32 @@
...
+    interface DragsterStatic {
+        (elem: HTMLElement): Dragster;
+        new (elem: HTMLElement): Dragster;
+    }
 }
 
-export = Dragster;
\ No newline at end of file
+// window.Dragster
+declare var Dragster: Dragster.DragsterStatic;
```
Comment (DanielRosenwasser) [inline dragster/index.d.ts]: Better way to do this and the below:

```ts
// Support through imports.
declare var Dragster: Dragster.DragsterStatic;
export = Dragster;

// Support as a global when not using modules.
export as namespace Dragster.
```

Comment (DanielRosenwasser) [inline dragster/index.d.ts]: Better way to do this and the below:

```ts
// Support through imports.
declare var Dragster: Dragster.DragsterStatic;
export = Dragster;

// Support as a global when not using modules.
export as namespace Dragster.
```

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: dragster/index.d.ts (hunk 2 above)
Comment (zskovacs) [inline dragster/index.d.ts]: Thanks, i fixed those.

File: dragster/tslint.json [hunk 3]
```diff
@@ -1 +0,0 @@
-{ "extends": "../tslint.json" }
```
Comment (aozgaa) [inline dragster/tslint.json]: Why did you remove `tslint.json`?

Comment (zskovacs) [inline dragster/tslint.json]: It failed because of the multiple module export. I saw some other library use that way. Anyway, I can put it back, @DanielRosenwasser solved that problem.
